schema.graphql

```diff
@@ -223,6 +223,37 @@ type Trader @entity {
     referrerCode: ReferralCode
     "the referral code the trader has registered to themselve"
     refereeCode: ReferralCode
+    "A list of markets that the traders have traded on"
+    traderMarkets: [TraderMarket!]! @derivedFrom(field: "traderRef")
+}
+
+type TraderMarket @entity {
```

Comment:
Do we need both `Position` and `TraderMarket`? It seems we only need one of them.

Reply:
Will merge them in the future.